exposed-java-time/src/test/kotlin/org/jetbrains/exposed/JavaTimeTests.kt

```diff
@@ -284,6 +284,42 @@ open class JavaTimeBaseTest : DatabaseTestsBase() {
         }
     }
 
+    @Test
+    fun testLocalDateTimeComparison() {
+        val testTableDT = object : IntIdTable("test_table_dt") {
+            val created = datetime("created")
+            val modified = datetime("modified")
+        }
+
+        withTables(testTableDT) { testDb ->
+            val mayTheFourthDT = LocalDateTime.of(2011, 5, 4, 13, 0, 21, 871130789)
+            val nowDT = LocalDateTime.now()
+            val id1 = testTableDT.insertAndGetId {
+                it[created] = mayTheFourthDT
+                it[modified] = mayTheFourthDT
+            }
+            val id2 = testTableDT.insertAndGetId {
+                it[created] = mayTheFourthDT
+                it[modified] = nowDT
+            }
+
+            // these DB take the nanosecond value 871_130_789 and round up to default precision (e.g. in Oracle: 871_131)
+            val requiresExplicitCast = listOf(TestDB.ORACLE, TestDB.H2_ORACLE, TestDB.H2_PSQL, TestDB.H2_SQLSERVER)
```

Comment:
nit: What do you think of sticking to one variable name in other files for consistency? `requiresExplicitCast` vs `requiresExplicitDTCast`